CodeFormatCore/src/Format/Analyzer/SemicolonAnalyzer.cpp

```diff
@@ -82,9 +83,9 @@ void SemicolonAnalyzer::AddSemicolon(LuaSyntaxNode n, const LuaSyntaxTree &t) {
     }
 }
 
-void SemicolonAnalyzer::InsertNewLineBeforeNode(LuaSyntaxNode n, const LuaSyntaxTree &t) {
-    auto token = n.GetFirstToken(t); // line breaks are put in front of the statement itself by non-first statements
-    if (token.IsToken(t)) {
+void SemicolonAnalyzer::InsertNewLineBeforeNextNode(LuaSyntaxNode n, const LuaSyntaxTree& t) {
```

Comment:
When I format, the & symbol will be close to t, isn't that your formatter like this?

Reply:
When I actively format the document, yes. There seem to be conflicting formats in my IDE, and I didn't remember to check those.